live-examples/js-examples/regexp/regexp-prototype-dotall.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">
+var regex1 = new RegExp('foo', 's');
```

Comment:
There's a blank line at the start of this example. The source should be like:

    <code id="static-js">const regex1 = new RegExp('foo', 's');

Weird I know.